libparsec/crates/types/src/invite.rs

```diff
@@ -26,14 +26,25 @@ pub enum InvitationType {
     Device,
 }
 
+#[derive(Debug, Clone)]
+pub struct InvitationTypeParseError(String);
```

Comment:
```suggestion
pub struct InvitationTypeParseError;
```

No need to display the invalid data

Reply:
done